tools/rosbash/roszsh

```diff
@@ -740,7 +739,7 @@ compctl -K "_roscomplete_rosbag" + -g "*.bag *(/)" "rosbag"
 compctl -K "_roscomplete_rosnode" "rosnode"
 compctl -K "_roscomplete_rosparam" "rosparam"
 compctl -x 'p[0,2]' -K "_roscomplete_rostopic" - 'n[1,/] p[3]' -K "_roscomplete_rostopic" - 'p[3]' -S ' ' -K "_roscomplete_rostopic" - 'p[4]' -Q -K "_roscomplete_rostopic" -- "rostopic"
-compctl -K "_roscomplete_rosservice" "rosservice"
+compctl -Q -K "_roscomplete_rosservice" "rosservice"
```

Comment:
-Q prevents zsh from escaping special characters like quotes and spaces in the completion candidates. Since rosmsg-proto already gives output suitable for parsing by the shell, escaping it just makes it illegible